docs/src/examples/components/Dropdown/Variations/DropdownExamplePosition.shorthand.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Dropdown, Grid, Alignment, Position } from '@stardust-ui/react'
+import { useSelectKnob, useBooleanKnob } from '@stardust-ui/docs-components'
+
+const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange']
+
+const DropdownExamplePosition = () => {
+  const [open] = useBooleanKnob({ name: 'open', initialValue: true })
+
+  const [positionAndAlign] = useSelectKnob<PositionAndAlign>({
+    name: 'position-align',
+    initialValue: 'below',
+    values: positionAndAlignValues,
+  })
+
+  const [position, align] = _.split(positionAndAlign, '-') as [Position, Alignment]
+
+  return (
+    <Grid columns="1" variables={{ padding: '140px 0' }} styles={{ justifyItems: 'center' }}>
+      <Dropdown
+        open={open}
+        items={inputItems}
+        placeholder={`Opens ${position} trigger${align ? ` aligned to ${align}` : ''}.`}
+        align={align}
+        position={position}
+      />
+    </Grid>
+  )
+}
+
+export default DropdownExamplePosition
+
+type PositionAndAlign =
```

Comment:
Just wonder, do we really need this type?

Reply:
Don't needed, just for safety. Placed it at the end so that it doesn't distract user. Can be removed if you feel strongly about it @layershifter

Reply:
As it was mentioned before, example should extra simple and contain less code. So if we don't have reasons to have this type, let's remove it.

Reply:
done, thanks